src/main/java/seedu/saveit/logic/commands/AddCommand.java

```diff
@@ -74,7 +74,7 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
         case DUMMY_STATEMENT: //adding solution to existing issue
             assert (toAdd.getDescription().getValue().equals(DUMMY_DESCRIPTION));
             assert (toAdd.getSolutions().size() == 1);
-            if (model.getCurrentDirectory().isIssueLevel()) {
+            if (!model.getCurrentDirectory().isRootLevel()) {
```

Comment:
because currently we allowed both issue level and solution level to edit or add solution. so as long as not root level, it is acceptable

Reply:
😯If so I recommend `isIssueLevel || isSolutionLevel.` I just feel if there are more levels `!isRootLevel` will cause a bug, because `!isRootLevel` is not it's 'real' condition. It just accidentally fits in here.
But currently this should be fine. If you want to leave it it's ok.

Reply:
isIssueLevel || isSolutionLevel is much clearly for sure